src/components/homePage/homePage.jsx

```diff
@@ -38,44 +46,33 @@ function HomePage() {
                     <TableCell align="right">Name&nbsp;</TableCell>
                     <TableCell align="right">Email&nbsp;</TableCell>
                     <TableCell align="right">Designation&nbsp;</TableCell>
-                    <TableCell align="right">TotalPoints</TableCell>
                     <TableCell align="right">COP Name&nbsp;</TableCell>
-                    <TableCell align="right">CreatedBy&nbsp;</TableCell>
-                    <TableCell align="right">UpdatedBy&nbsp;</TableCell>
-                    <TableCell align="right">Total Experience&nbsp;</TableCell>
+                    <TableCell align="right">Action&nbsp;</TableCell>
                 </TableRow>
                 </TableHead>
                 <TableBody>
                 {members.items && members.items.map((row) => (
                     <TableRow key={row._id}>
-                    <TableCell component="th" scope="row">
-                        {row._id}
-                    </TableCell>
-                    <TableCell align="right">{row._id}</TableCell>
-                    <TableCell align="right">{row.name}</TableCell>
-                    <TableCell align="right">{row.email}</TableCell>
-                    <TableCell align="right">{row.designation}</TableCell>
-                    <TableCell align="right">{row.totalPoints}</TableCell>
-                    <TableCell align="right">{row.copName}</TableCell>
-                    <TableCell align="right">{row._createdBy}</TableCell>
-                    <TableCell align="right">{row._updatedBy}</TableCell>
-                    <TableCell align="right">{row.totalExperience}</TableCell>
+                        <TableCell align="right">{row._id}</TableCell>
+                        <TableCell align="right">
+                            <Link to={`/member/${row._id}`}>Detail{row.name}</Link>
+                        </TableCell>
+                        <TableCell align="right">{row.email}</TableCell>
+                        <TableCell align="right">{row.designation}</TableCell>
+                        <TableCell align="right">{row.copName}</TableCell>
+                        <TableCell align="right">
+                            <div className={classes.root}>
```

Comment:
If div is being used as container, then we can use Box from material-ui and use it like this:
`<Box component="div">`
Similarly for span, we can use:
`<Box component="span">`

https://material-ui.com/components/box/